app/models/import_service_config.rb

```diff
@@ -15,7 +16,9 @@ module ImportServiceConfig
 
   ALLOWED_SCP_MODELS = [Study, StudyFile].freeze
 
-  attr_accessor :client, :user_id, :file_id, :study_id, :branding_group_id
+  DEFAULT_OBSM_KEYS = %w[X_umap].freeze
```

Comment:
the default key in ingest_pipeline.py is X_tsne. This default would just override that one but it might be clearer to have the same default in both places. I'll make a ticket to update ingest_pipeline.py to X_umap.

Reply:
Oh right, I'd forgotten about that.  I don't have a vested interest in it being `X_umap`, but that was what I used in testing for the HCA file.  I didn't even check if it had `X_tsne`.  I agree they should be the same.